src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -17,6 +17,11 @@
 
 // This is a generated sample ("FlattenedAsync", "empty_response_type_with_response_handling")
 
+// sample-metadata
+//   title: Test response handling for methods that return empty
+//   description: Test response handling for methods that return empty
+//   usage: dotnet run
```

Comment:
Is it necessary to call out the main class here as well, similar to Java?
e.g., `dotnet run /p:StartupObject=Namespace.MainClass`?